packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -225,48 +248,64 @@ const useTransactionHistoryWithoutStatuses = (
     cctpTransfersTestnet.isLoadingDeposits ||
     cctpTransfersTestnet.isLoadingWithdrawals
 
+  const fetcher = useCallback(
+    (type: 'deposits' | 'withdrawals') => {
+      const fetcherFn = type === 'deposits' ? fetchDeposits : fetchWithdrawals
+
+      return Promise.all(
+        multiChainFetchList
+          .filter(chainPair => {
+            if (isTestnetMode) {
+              // in testnet mode we fetch all chain pairs
+              return chainPair
+            }
+            // otherwise don't fetch testnet chain pairs
+            return !isTestnetChainPair(chainPair)
```

Comment:
Let's just inline the logic from `isTestnetChainPair`, no need for extra indirection

```suggestion
            if (isTestnetMode) {
              // in testnet mode we fetch all chain pairs
              return chainPair
            }
            // otherwise don't fetch testnet chain pairs
            return isNetwork(chainPair.parentChain).isTestnet
```

Reply:
updated